### PR TITLE
Make the link icon light in dark mode and dark in light mode

### LINKED ISSUE #903: Use filter to make the link icon light in dark mode.
#### Description
Use filter to make the link icon light in dark mode.
Dropdown-Link-Icon-Color

#### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by yashhhguptaaa**
**Description**

This PR fixes #903 

## Changes
Make the link icon light in dark mode and dark in light mode

## Screenshot
https://user-images.githubusercontent.com/82795074/206566260-6c9900a2-3e17-4273-9b21-9b2bd9ad09ce.mp4

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -35,8 +35,7 @@
 							<a href="{{sublink.link}}" {% if sublink.new_window %}target="_blank" {% endif %}
 							class="subnav-item">{{sublink.name}}
 							{% if sublink.external %}
-							<img src="/images/buttons/external-link_gray.svg"
-							style="max-width:  10px; vertical-align:top;" />
+							<img src="/images/buttons/external-link_gray.svg" class="nav-sublink-logo">
```
Comment (Nikhil-Ladha) [inline _includes/navigation.html:38]: Always add the `alt` attributes` to the `img` tags.

Comment (yashhhguptaaa) [reply _includes/navigation.html]: @Nikhil-Ladha I actually forgot about this open PR and your comment. I have added the alt attributes.
you can review now.

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
Please follow the steps mentioned [here](https://github.com/meshery/meshery.io/pull/933/checks?check_run_id=10165519202) to fix the DCO issue.

**Comment by UtkarshMishra12** [comment]
@yashhhguptaaa  DCO Failing 
Please follow the steps mentioned here [link](https://github.com/meshery/meshery.io/pull/933/checks?check_run_id=10301396021)

or 

To add your Signed-off-by line to every commit in this branch:

Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
In your local branch, run: **git rebase HEAD~4 --signoff**
Force push your changes to overwrite the branch: **git push --force-with-lease origin ci/yashhhguptaaa/903**

**Comment by leecalcote** [comment]
@yashhhguptaaa still with us?

**Comment by aryans1319** [comment]
Can I work on this issue if it's not fixed ? @leecalcote @UtkarshMishra12

**Comment by UtkarshMishra12** [comment]
@Chadha93 We can merge this PR by resolving the DCO issue.  // @Nikhil-Ladha

**Pull request merged**